ios-boilerplate/Network/WebServiceRepositories/ExampleServiceAdapter.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ExampleServiceAdapter.swift
+//  ios-boilerplate
+//
+//  Created by developer on 29.12.2020.
+//  Copyright © 2020 Adesso Turkey. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+protocol ExampleServiceAdapterProtocol {
+    func exampleRequest() -> Single<ExampleResponse?>
+}
+
+class ExampleServiceRepository: ExampleServiceAdapterProtocol {
```

Comment:
In sample application adapter logic is replaced with use case. So to be consistent, either let's change this to use case, or exclude adapter logic from this scope.